dash/extract-meta.js

```diff
@@ -37,7 +37,7 @@ if (!src.length) {
 }
 
 if (fs.existsSync('tsconfig.json')) {
-    tsconfig = JSON.parse(fs.readFileSync('tsconfig.json'));
+    tsconfig = JSON.parse(fs.readFileSync('tsconfig.json')).compilerOptions;
```

Comment:
Just remembered some 3rd party tools that would wrap around this (I'm thinking like NX which i use alot) actually uses Jsonc. Can we change out the JSON.parse to support this? Or add another ticket